src/PhpSpreadsheet/Writer/Ods/Cell/Style.php

```diff
@@ -78,6 +80,80 @@ private function writeFillStyle(Fill $fill): void
         }
     }
 
+    private function writeBordersStyle(Borders $borders): void
+    {
+        $this->writeBorderStyle('bottom', $borders->getBottom());
+        $this->writeBorderStyle('left', $borders->getLeft());
+        $this->writeBorderStyle('right', $borders->getRight());
+        $this->writeBorderStyle('top', $borders->getTop());
+    }
+
+    private function writeBorderStyle(string $direction, Border $border): void
+    {
+        if ($border->getBorderStyle() === Border::BORDER_NONE) {
+            return;
+        }
+
+        $this->writer->writeAttribute('fo:border-' . $direction, sprintf(
+            '%s %s #%s',
+            $this->mapBorderWidth($border),
+            $this->mapBorderStyle($border),
+            $border->getColor()->getRGB(),
+        ));
+    }
+
+    protected function mapBorderWidth(Border $border): string
```

Comment:
```suggestion
    private function mapBorderWidth(Border $border): string
```

all new method should be private instead of protected

Reply:
Fixed